src/authV2/azext_authV2/custom.py

```diff
@@ -821,6 +918,15 @@ def update_openid_connect_provider_settings(cmd, resource_group_name, name, prov
         registration["clientId"] = client_id
     if client_secret_setting_name is not None:
         registration["clientCredential"]["clientSecretSettingName"] = client_secret_setting_name
+    if client_secret is not None:
+        provider_name_prefix = provider_name.upper()
```

Comment:
We seem to repeat this logic in both update and add. I'm curious if we could extract into a helper method?